components/builder-db/src/migrations/2020-03-27-193418_origin_members_dupe_fixes/up.sql

```diff
@@ -0,0 +1,24 @@
+DELETE FROM
+  origin_members T1
+WHERE
+  EXISTS (
+    SELECT
+      1
+    FROM
+      origin_members T2
+    WHERE
+      T1.account_id = T2.account_id
+      AND T1.origin = T2.origin
+      AND T1.ctid > T2.ctid
```

Comment:
Maybe tiebreak to preserve oldest first; test T1.created_at > T2.created_at and if there are entries with duplicate timestamps can tiebreak on CTID>

Reply:
Agreed @markan. I added your suggesting in https://github.com/habitat-sh/builder/pull/1374/commits/bf3131246c1c26271c091ea2109b70560d45399e